tools/dev-doctor/etcnfsconfcheck.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/ini.v1"
+)
+
+// An etcdNFSConfCheck checks /etc/nfs.conf.
+type etcNFSConfCheck struct{}
+
+func (etcNFSConfCheck) id() string {
+	return "/etc/nfs.conf"
+}
+
+func (etcNFSConfCheck) run() (checkResult, string) {
+	data, err := ioutil.ReadFile("/etc/nfs.conf")
+	switch {
+	case os.IsNotExist(err):
+		return checkError, "/etc/nfs.conf does not exist"
```

Comment:
NFS seems to work fine on my setup without this file in place 🤔

Reply:
Done. NFS checks are now opt-in.